fixture/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+find_path(CSM_INCLUDE_DIR NAMES "csm.h"
+            PATH_SUFFIXES "csm" 
+            PATHS $ENV{CONDA_PREFIX}/include/)
+find_library(CSM_LIBRARY csmapi PATHS $ENV{CONDA_PREFIX}/lib)
```

Comment:
Someone should write a findcsm script for cmake if the cmake build system is going to become standard.